### ISSUE TITLE
OCHLV data is pulled only for date range ending on February 6th

### ISSUE THREAD

**Issue by 1J0A6L6** [ccxt/ccxt#17567]
### Operating System

MacOS Big Sur

### Programming Languages

python

### CCXT Version

3.0.66

### Description

Objective: pull OCHLV data from Binance into an algorithm, prepare the data as a dataframe, and have it start with the most recent value (current candle) first, and have the times be accurate. 

My issue: the CCXT is returning the same interval (ending data in February 6, 2023 at 4:55pm, and starting sometime before that) no matter how long I wait. It's about an hour since I launched the code, and it's still giving me data from 4:55pm as the most recent. It's 10:50pm here, and I started running this code at around 9:50pm Eastern Standard/Daylight Time.
It seems like the CCXT library is giving me the same exact data no matter when I submit the code, which won't be useful for analysis of current data as time goes on.

My code: 

# Retrieve the OCHLV data 5m
ohlcv_data_5min = exchange.fetch_ohlcv(symbol, '5m')

# Print the OCHLV data
print(f'OCHLV Data 5 min \n')
for data in ohlcv_data_5min:
    timestamp, open_price, high_price, low_price, close_price, volume = data
    #print(f'Timestamp: {timestamp}, Open: {open_price}, High: {high_price}, Low: {low_price}, Close: {close_price}, Volume: {volume}')

# Convert the OHLCV data into a Pandas DataFrame
df5min = pd.DataFrame(ohlcv_data_5min, columns=['timestamp', 'open', 'high', 'low', 'close', 'volume'])

# Convert the timestamp column to a datetime object
df5min['timestamp'] = pd.to_datetime(df5min['timestamp'], unit='ms')

# Set the timestamp column as the index
df5min.set_index('timestamp', inplace=True)
df5min.sort_values(by='timestamp', ascending=False, inplace=True)

# Print the resulting DataFrame
print(df5min)

print(f'\n')

### Code

```
  

```
# Retrieve the OCHLV data 5m
ohlcv_data_5min = exchange.fetch_ohlcv(symbol, '5m')

# Print the OCHLV data
print(f'OCHLV Data 5 min \n')
for data in ohlcv_data_5min:
    timestamp, open_price, high_price, low_price, close_price, volume = data
    #print(f'Timestamp: {timestamp}, Open: {open_price}, High: {high_price}, Low: {low_price}, Close: {close_price}, Volume: {volume}')

# Convert the OHLCV data into a Pandas DataFrame
df5min = pd.DataFrame(ohlcv_data_5min, columns=['timestamp', 'open', 'high', 'low', 'close', 'volume'])

# Convert the timestamp column to a datetime object
df5min['timestamp'] = pd.to_datetime(df5min['timestamp'], unit='ms')

# Set the timestamp column as the index
df5min.set_index('timestamp', inplace=True)
df5min.sort_values(by='timestamp', ascending=False, inplace=True)

# Print the resulting DataFrame
print(df5min)

print(f'\n')

**Comment by pcriadoperez** [comment]
Hi @1J0A6L6 , I just tested `fetchOHLCV` and it is giving me the latest data.

I ran this at time: 1681647586
```python
binance.fetchOHLCV(BTC/USDT,5m,None,5)
[[1681646100000, 30247.71, 30253.57, 30220.4, 30226.45, 103.78069],
 [1681646400000, 30226.46, 30239.65, 30220.09, 30238.44, 91.8392],
 [1681646700000, 30238.44, 30238.45, 30237.25, 30237.43, 53.99046],
 [1681647000000, 30237.42, 30254.65, 30237.42, 30254.65, 48.31371],
 [1681647300000, 30254.64, 30277.38, 30254.64, 30270.12, 61.81469]]
 ```
 
 Also I was a bit confused about February 6? Are you trying to fetch data before february 6?
 
 If you are trying to grab data in the past, it will always return the same result, if you wish to move the timeframe window you can adjust the `since` and `limit` values to paginate across the data.

**Comment by 1J0A6L6** [comment]
I tried the same code on Coinbase and it works. I don't have login credentials for Binance, but didn't think I'd need to for the public data.

**Comment by pcriadoperez** [comment]
Yes, for binance you don't need keys for public data. Can you share the code you are using to try and reproduce it?

**Comment by ttodua** [comment]
closing stale issue. let us know if any issues
